pkg/mdformatter/linktransformer/config.go

```diff
@@ -20,6 +21,9 @@ type Config struct {
 	Version int
 
 	Validators []ValidatorConfig `yaml:"validators"`
+	Timeout    string            `yaml:"timeout"`
```

Comment:
Let's start with that, if needed we could do also per validator with this being default.